tests/cmake_integration/generated_global/idl/CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+# Distributed under the OpenDDS License. See accompanying LICENSE
+# file or http://www.opendds.org/license.html for details.
+#
+# This file is setup the way it is for testing purposes. For a cleaner example
+# see DevGuideExamples/DCPS/Messenger/CMakeLists.txt
+
+project(opendds_generated_property CXX)
+cmake_minimum_required(VERSION 3.8.2)
+
+find_package(OpenDDS REQUIRED)
+
+
+set(CMAKE_CXX_COMPILER ${OPENDDS_COMPILER})
+
+if (NOT OPENDDS_OWNERSHIP_PROFILE)
+  message(FATAL_ERROR "These tests require OpenDDS to be built with Ownership Profile enabled")
+endif()
+
+set(target_prefix "idl_")
```

Comment:
Make `target_prefix` unique like the other tests. When using `tests/cmake_integration/CMakeLists.txt` all the targets are in the same set.